x11-wm/enlightenment/enlightenment-0.21.8.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+MY_P=${P/_/-}
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_SUB_PROJECT="core"
+	EGIT_URI_APPEND="${PN}"
+else
+	SRC_URI="https://download.enlightenment.org/rel/apps/${PN}/${MY_P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~x64-solaris ~x86-solaris"
+fi
+
+inherit enlightenment
+
+DESCRIPTION="Enlightenment DR17 window manager"
+
+LICENSE="BSD-2"
+SLOT="0.17/${PV%%_*}"
+
+__CONF_MODS=(
+	applications bindings dialogs display
+	interaction intl menus
+	paths performance randr shelves theme
+	window-manipulation window-remembers
+)
+__NORM_MODS=(
+	appmenu backlight bluez4 battery
+	clock conf connman cpufreq everything
+	fileman fileman-opinfo gadman geolocation
+	ibar ibox lokker
+	mixer msgbus music-control notification
+	pager packagekit pager-plain policy-mobile quickaccess
+	shot start syscon systray tasks teamwork temperature tiling time
+	winlist wireless wizard wl-desktop-shell wl-drm wl-text-input
+	wl-weekeyboard wl-wl wl-x11 xkbswitch xwayland
+)
+IUSE_E_MODULES=(
+	${__CONF_MODS[@]/#/enlightenment_modules_conf-}
+	${__NORM_MODS[@]/#/enlightenment_modules_}
+)
+
+IUSE="pam spell static-libs systemd ukit wayland ${IUSE_E_MODULES[@]/#/+}"
+
+RDEPEND="
+	pam? ( sys-libs/pam )
+	systemd? ( sys-apps/systemd )
+	wayland? (
+		dev-libs/efl[wayland]
+		>=dev-libs/wayland-1.10.0
```

Comment:
`>=dev-libs/wayland-1.11.0` is actually needed since https://git.enlightenment.org/core/enlightenment.git/?id=9c23fbaf280e8c24ce02f2bf0d38d4fb0e9d12b1